Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -30,30 +30,33 @@ function App() {
     <AppContext.Provider value={Harvest}>
       <Route exact path="/" component={Home} />
       <Route exact path="/home/spa" component={Spa} />
-      <Route path="/request/create/:projectId?" component={RequestContainer} />
-      <Route path="/request/approve/:projectId" component={ApprovalContainer} />
-      <Route
+      <ConditionalRoute roles={["FieldManager"]} path="/request/create/:projectId?" component={RequestContainer} />
+      <ConditionalRoute roles={["FieldManager"]} path="/request/approve/:projectId" component={ApprovalContainer} />
+      <ConditionalRoute
+        roles={["FieldManager"]}
         path="/request/changeAccount/:projectId"
         component={AccountChangeContainer}
       />
       <Route
         path="/project/invoices/:projectId"
         component={InvoiceListContainer}
       />
-      <Route
+      <ConditionalRoute
+        roles={["FieldManager"]}
         path="/invoice/details/:invoiceId"
         component={InvoiceDetailContainer}
       />
-      <ConditionalRoute roles={['FieldManager', 'Supervisor']} path="/quote/create/:projectId" component={QuoteContainer} />
+      <ConditionalRoute roles={["FieldManager"]} path="/quote/create/:projectId" component={QuoteContainer} />
```

Comment:
supervisors need to be able to create a quote too.  Probably they shouldn't be able to submit them, but that's a separate question

Reply:
Fixed